src/utils/generateSlackFormatterUrl.js

```diff
@@ -1,15 +1,19 @@
-const querystring = require('querystring');
+const tinyurl = require('tinyurl');
 
-const generateSlackFormatterUrl = attachments => {
-  const baseSlackUrl = 'https://api.slack.com/docs/messages/builder?';
+const generateSlackFormatterUrl = async (blocks, config) => {
+  const baseSlackUrl = `https://app.slack.com/block-kit-builder/${config.get('slack:orgId')}#`;
+  let url;
 
-  const attachmentsToStringify = JSON.stringify({
-    attachments,
-  });
+  const stringifiedBlocks = JSON.stringify(blocks);
+  const stringifiedQuery = encodeURIComponent(stringifiedBlocks);
 
-  const stringifiedQuery = querystring.stringify({ msg: attachmentsToStringify });
+  const longUrl = `${baseSlackUrl}%7B"blocks":${stringifiedQuery}%7D`;
 
-  return `${baseSlackUrl}${stringifiedQuery}`;
+  if (process.env.ENABLE_TINY) {
+    url = await tinyurl.shorten(longUrl);
```

Comment:
Is it possible to do this without `tinyurl`? Do you think some orgs may not want their links going through this external service?

Reply:
Yes, that is why I require it to be enabled. Do you think we should remove it entirely?

Reply:
@therynamo Seems like an acceptable opt-in feature. Were worried about running into URL length limits or something?

Reply:
![image](https://user-images.githubusercontent.com/8508556/96353465-0039f500-1092-11eb-94ac-50eef635e857.png)
It is partly that, but also... this is my console after a "small" message gets released. If you want to copy paste that... it isn't great. And the url breaks up with the uri encoding, so it is not `CMD+Click`able.